packages/archway-proto/__tests__/messages.test.js

```diff
@@ -0,0 +1,43 @@
+import { coin } from '@cosmjs/amino';
```

Comment:
Moving this test to the `src` folder is better. Let's keep the same name convention as in `arch3-core`, using the `{component}.spec.ts` for tests.

Reply:
That was an existing file that came with the project. I'll delete it. But I want to confirm am I writing tests for every single file under the archway folder?

Reply:
I don't think this will work. As a test I created an event.spec.ts file next to the file src/codegen/archway/rewards/v1beta1/events.ts and when I ran the codegen again it deleted the spec file. Meaning it probably deletes all the file in that location and adds fresh